views/styles/graph_page_styles.js

```diff
@@ -8,4 +15,18 @@ export const graphPageStyles = (theme) => ({
   configForm: { width: '45%' },
   configFormControl: { width: '100%' },
   configLabel: { marginTop: '25px', marginLeft: '30px' },
+  graphToolbar: {
```

Comment:
I wonder if embracing styles-in-js is the way to go with this app. It doesn't seem avoidable given the way the Material Design library expects style props.